static/app/views/dashboards/widgetBuilder/utils.tsx

```diff
@@ -295,15 +298,25 @@ export function normalizeQueries({
 
   if (DisplayType.BIG_NUMBER === displayType) {
     // For world map chart, cap fields of the queries to only one field.
```

Comment:
Can we also remove this comment? It's pretty old and doesn't describe this part of the code anymore at all